### DISCUSSION TITLE
cmi.viewrange permission doesn't work

### DISCUSSION THREAD

**Discussion by Fiks75** [Zrips/CMI#2528] [Q&A]
**Question: **
How to limit viewrange if cmi.viewrange permission doesn't work?
Example, permission cmi.viewrange.5 does not limit viewing to 5 chunks.

**Comment by mrfloris** [comment]
It kinda sucks that enough has changed since 2019 that 1.17.1 currently doesn't support this feature in CMI. Hopefully it will return with 1.18. (sorry for bumping old discussion)
